docs/platform/howto/change-billing-contact.rst

```diff
@@ -0,0 +1,28 @@
+Change billing contact
+======================
+
+For projects using credit card billing, the billing contact is automatically set to the user who owns the project's credit card. This means that the billing contact is set to the user who added the payment card and assigned it to project(s). You cannot remove the billing contact from the project. However, you change the billing contact user. 
+
+To change the billing contact user, you need to add a new payment card to the user account you want to assign as the billing contact user. The new payment card can be the same as the one currently used. 
+
+
+Add a new payment card
```

Comment:
I agree! This is a bit of a mixed scenario. To change the billing contact, you need to add another user as the billing contact, and this can be done by adding the payment card to the user who'd be the new billing contact. 

I think we need a new article that provides the user with some background info on Billing contact and all the operations the user can perform, like add new billing contact, edit/update/remove billing contact, add a new payment card, edit/update/remove payment card(as subheadings). 

But since the new topic currently doesn't exist, should we keep this as is now and create a new article once all the migration is in place?

Reply:
I think we should still split the topics so that the information we do have (how to change a contact, how to add a card) is well-organised in the current structure and we don't lose anything. Then we can flesh this out later but we've still got what we already have, if that makes sense.

Reply:
So this will be two articles: 
- HowTo add a new payment card 
- HowTo Change billing contact

Reply:
I will split the topics into two. Just an FYI, this entire feature is changing at the end of August.